src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -178,8 +183,37 @@ public void startup() {
         reportState(State.RUNNING);
     }
 
+    private void initializeMqttClientKeyStore() throws
+            CsrProcessingException, CsrGeneratingException, KeyStoreException, ServiceLoadException {
+        mqttClientKeyStore.init();
+
+        certificateAuthoritiesTopic = kernel
+                .locate(ClientDevicesAuthService.CLIENT_DEVICES_AUTH_SERVICE_NAME)
+                .getConfig()
+                .lookup(RUNTIME_STORE_NAMESPACE_TOPIC,
+                        ClientDevicesAuthService.CERTIFICATES_KEY,
+                        ClientDevicesAuthService.AUTHORITIES_TOPIC);
+        onCertificateAuthoritiesChange = (why, newv) -> {
+            try {
+                List<String> caPemList = (List<String>) newv.toPOJO();
+                if (Utils.isEmpty(caPemList)) {
+                    return;
+                }
+                mqttClientKeyStore.updateCA(caPemList);
+            } catch (IOException | CertificateException | KeyStoreException e) {
+                serviceErrored(e);
+            }
+        };
+
+        certificateAuthoritiesTopic.subscribe(onCertificateAuthoritiesChange);
```

Comment:
`initializeMqttClientKeyStore` method doing more than the name suggests. Please break it down.